src/mailroom.py

```diff
@@ -0,0 +1,93 @@
+"""Automate thankyou notes and generates donor reports."""
+
+import io
+import sys
+from sys import version_info
+
+from builtins import input
+
+donors = {
+    'Matthew Wade': [50.00, 100.00],
+    'Betty Stewart': [100.00, 50.00, 150.00],
+    'Timothy Berg': [10.00],
+    'Paul Benson': [5]}
+
+
+def get_input():
+    """Function gathers all user input and controls program flow."""
+    # Comply inputs between Python 2 and 3. Default to Python 3:
+    user_input = input
+    # If Python 2:
+    if sys.version_info[:2] <= (2, 7):
+        user_input = raw_input
+    ask_report = user_input("Thank you letter (1), report (2), return to the home menu, (3), or quit? (q) ")
```

Comment:
A better approach would be to make your prompts global constants. per Python convention, global constants are written in ALL_CAPS, are kept near the top of your module in the global scope, and are never modified by the program.